core/drivers/crypto/crypto_api/cipher/cipher.c

```diff
@@ -0,0 +1,201 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright 2018-2019 NXP
+ *
+ * Brief   Crypto Cipher interface implementation to enable HW driver.
```

Comment:
Please drop "Brief" here and in other files.

Reply:
Ok